src/main/java/com/faforever/client/remote/FafServerAccessorImpl.java

```diff
@@ -247,15 +250,17 @@ protected Void call() throws Exception {
 
             logger.info("FAF server connection established");
             Platform.runLater(() -> connectionState.set(ConnectionState.CONNECTED));
+            reconnectService.resetConnectionFailures();
 
             blockingReadServer(fafServerSocket);
           } catch (IOException e) {
             Platform.runLater(() -> connectionState.set(ConnectionState.DISCONNECTED));
             if (isCancelled()) {
               logger.debug("Connection to FAF server has been closed");
             } else {
-              logger.warn("Lost connection to FAF server, trying to reconnect in " + RECONNECT_DELAY / 1000 + "s", e);
-              Thread.sleep(RECONNECT_DELAY);
+              logger.warn("Lost connection to Server", e);
```

Comment:
Can we switch to `@Slf4J` in this class?

Reply:
ok